examples/import-sample-export-gnn.ipynb

```diff
@@ -141,7 +141,7 @@
    "source": [
     "from graphdatascience.server_version.server_version import ServerVersion\n",
     "\n",
-    "assert gds.server_version() >= ServerVersion(2, 2, 0)"
+    "assert gds.server_version() >= ServerVersion(2, 6, 0)"
```

Comment:
```suggestion
    "assert gds.server_version() >= ServerVersion(2, 5, 0)"
```
I think 2.5 is enough to cover `gds.graph.relationships.stream` (`2.4` is enough for `rwr`). Don't feel strongly about it though.